<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QDialog" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_pic3">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>-5</y>
     <width>931</width>
     <height>621</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>331</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="choose_chapter">
      <property name="text">
       <string>Choose Chapter</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="search_with_words">
      <property name="text">
       <string>Search With Words</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="see_discussions">
      <property name="text">
       <string>See Discussions</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="send_friend_request">
      <property name="text">
       <string>Send Friend Request</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="so_exit">
      <property name="text">
       <string>Sign out or Exit LearNet</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>501</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Choose a chapter and start studying</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Choose a chapter based on a few words</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>See other users' discussions based on a certain chapter  and post a message</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Send a friend request to another user - see your friends list or a list with all users</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Sign out or exit application</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
